mcs/class/Makefile

```diff
@@ -469,14 +469,14 @@ lib/$(monolite_dir)/Facades:
 
 $(monolite_files): | lib/$(monolite_dir)
 $(monolite_files): | lib/$(monolite_dir)/Facades
-$(monolite_files): lib/$(monolite_dir)/%: lib/build-$(HOST_PLATFORM)/%
```

Comment:
this doesn't look right, we do want a monolite with the platform-specific BCL files